randovania/games/dread/json_data/Burenia.txt

```diff
@@ -2422,11 +2431,11 @@ Extra - asset_id: collision_camera_021
   > Pickup (Missile+ Tank)
       Trivial
   > Tile Group (POWERBEAM) 1
-      Simple IBJ or Use Spin Boost
+      Trivial
```

Comment:
I would argue that this is not really trivial. You have to stop the screw attack manually so that you don´t destroy the floor you want to stand on. Maybe beginner movement if you don´t have any Jump items.

Reply:
Irrelevant. That would lock going up this way behind a trick. Everything has to be doable trickless. You have SA no matter what here, so you would have to cancel regardless of what you do unless you ibj up which is also a trick. Speed would be locked behind conservation beginner. Even so, the only trickless way to get all the way up is with Space Jump anyway.

Reply:
Fair enough